recycleIT-frontend/src/Components/Main/Main.tsx

```diff
@@ -18,98 +20,136 @@ const Main = () => {
     
     const handleChange = (event: React.SyntheticEvent, newValue: string) => {
         setValue(newValue);
-
       };
       const handleClick = () => {
         ref.current?.scrollIntoView({behavior: 'smooth'});
       };
     
-   const ButtonData =[
-    {
-        title:'PLASTIC',
-        id: 'plastic',
-        value: '1'
-    },
-    {
-        title:'PAPERE',
-        id: 'paper',
-        value: '2'
-    },
-    {
-        title:'GLASS',
-        id: 'glass',
-        value: '3'
-    },
-    {
-        title:'METAL',
-        id: 'metal',
-        value: '4'
-    },
-    {
-        title:'ELECTRONIC',
-        id: 'electronic',
-        value: '5'
-    }
-   ]
+    const ButtonData = [
+        {
+            title:'PLASTIC',
+            id: 'plastic',
+            value: '1'
+        },
+        {
+            title:'PAPER',
+            id: 'paper',
+            value: '2'
+        },
+        {
+            title:'GLASS',
+            id: 'glass',
+            value: '3'
+        },
+        {
+            title:'METAL',
+            id: 'metal',
+            value: '4'
+        },
+        {
+            title:'ELECTRONIC',
+            id: 'electronic',
+            value: '5'
+        }
+    ]    
     return (
         <>
             <Header/>
```

Comment:
it's odd that the header is added every time in the page component. Normally the structure is following:
```
<Header />
<Content>
    // routing logic
</Content>
<Footer />
```

and your page components are initialized from the router, while header and footer remain unchanged. This allows to not rerender header and footer every time you go from one page to another

Reply:
I have fixed the overlapping, but that is going to be in the newt PR.